packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -31,6 +31,28 @@ import { createContainerRuntimeFactoryWithDefaultDataStore } from "./testContain
 
 const summarizerClientType = "summarizer";
 
+/**
+ * This function should ONLY be used for back compat purposes
+ * LTS versions of the Loader/Container will not have the "getEntryPoint" method, so we need to fallback to "request"
+ * This function can be removed once LTS version of Loader moves to 2.0.0-internal.7.0.0
```

Comment:
> can be removed once LTS version [](http://example.com/codeflow?start=16&length=32)

this method should log a warning about LTS version being lower than 2.0.0-internal.7.0.0 otherwise it's easy to miss out that this needs to be removed.

Reply:
I am not sure a warning would help. When these tests run, any logs from it are not monitored unless the test fails.
You are right that these cases can be missed and they keep happening. I don't know if there is currently a way to track them. The only thing I can think of is to come up with some convention of marking these instances that should be taken care of when changing the LTS.

Reply:
you are right, warning is not the solution. 

I like your idea though, having a way to tag tests that need fixing in future and using a tool to scan our repo for these tags to open workitems against test owners or common tech-debt pool. 

If there are many tests/test-code that need update once LTS version moves to internal7.0; may be we should start adding that tag now or at least open a Feature and start parking Tasks! We can broadcast the same guidance for everyone. What do you think?

Reply:
There are places other than tests too that have to be updated when LTS updates. Actually, that is true to regular version updates too. We add back-compat code that should be removed when a version changes.
We can start a tagging process for it. There may be some convention that some team in already following. We used to add `back-compat <version>` but it was never consistent. Let's talk to other folks in the team.

Reply:
Cool. Test council would be a good place to start for the tagging idea. Thanks